api/app/signals/apps/api/v1/views/pdf.py

```diff
@@ -59,18 +63,41 @@ class GeneratePdfView(SingleObjectMixin, PDFTemplateView):
 
     def get_context_data(self, **kwargs):
         self.object = self.get_object()
-        rd_coordinates = self.object.location.get_rd_coordinates()
-        bbox = '{},{},{},{}'.format(
-            rd_coordinates.x - 340.00,
-            rd_coordinates.y - 125.00,
-            rd_coordinates.x + 340.00,
-            rd_coordinates.y + 125.00,
-        )
-
         logo_src = _get_data_uri(settings.API_PDF_LOGO_STATIC_FILE)
-
+        img_data_uri = None
+        bbox = None
+        x = 0
+        y = 0
```

Comment:
Make these `xp` and `yp` or even `top` and `left`. Naming as-is is a bit confusing becaus in this PR the `x` and `y` mean either the map tile location/index[1] or pixel location of marker on map.


[1] the `x` and `y` in `<SOME TILE SERVER URL>/{zoom}/{x}/{y}.png`.

Reply:
renamed